# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserBridgeParent.cpp
# Commit: 36b0646e8649
# Full Hash: 36b0646e86498662e770756ad54b1fd648a82184
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1620594
# File Overlap Count: 1
# Description:
#   Bug 1620594 - Part 7: Remove TabGroup and SystemGroup. r=nika,bas
#   
#   TabGroup never really made any difference in which thread something go
#   dispatched to. This was the intended use, but development of TabGroups
#   with abstract main threads never made it that far. The good thing is
# ==============================================================================

diff -r a5ad3b179c92 -r 36b0646e8649 dom/ipc/BrowserBridgeParent.cpp
--- a/dom/ipc/BrowserBridgeParent.cpp	Tue Apr 07 15:17:14 2020 +0000
+++ b/dom/ipc/BrowserBridgeParent.cpp	Tue Apr 07 15:17:47 2020 +0000
@@ -86,7 +86,7 @@
 
   // Tell the content process to set up its PBrowserChild.
   bool ok = aContentParent->SendConstructBrowser(
-      std::move(childEp), std::move(windowChildEp), aTabId, TabId(0),
+      std::move(childEp), std::move(windowChildEp), aTabId,
       tabContext.AsIPCTabContext(), aWindowInit, aChromeFlags,
       aContentParent->ChildID(), aContentParent->IsForBrowser(),
       /* aIsTopLevel */ false);